dataprofiler/profilers/profile_builder.py

```diff
@@ -1157,7 +1157,15 @@ def load(cls, filepath: str) -> BaseProfiler:
         """
         # Load profile from disk
         with open(filepath, "rb") as infile:
-            data: dict = pickle.load(infile)
+
+            if filepath.endswith(".json"):
```

Comment:
let's also allow for a user to specify the file_type like we do with save since we give the option for someone to technically use any extension given a type.

Reply:
agree -- parallel API for `save()` and `load()` in terms of accepted params